docs/metrics/getting-started-prometheus-grafana/Program.cs

```diff
@@ -31,9 +32,12 @@ public static void Main()
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
             .AddMeter("MyCompany.MyProduct.MyLibrary")
-            .AddPrometheusExporter(options => { options.StartHttpListener = true; })
+            .AddPrometheusExporter()
             .Build();
 
+        using var listener = new PrometheusHttpListener(meterProvider);
+        listener.Start();
+
```

Comment:
This seems a bit complicated than it has to be. Is it possible to make it work with just a single line (e.g. `meterProvider.AddPrometheusHttpListener()`)?

The object lifecycle management also seems complicated, e.g. what if the exporter got disposed before the listener?

Reply:
Updated based on the suggestion; please check [it](https://github.com/open-telemetry/opentelemetry-dotnet/blob/be87d4478b016e749bc7641e28db6c18756815fa/src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusExporterHttpListenerMeterProviderBuilderExtensions.cs) out.